mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,19 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
 
-  if (isHappy) {
+  if (isHappy === true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
 function greaterThan10(num) {
```

Comment:
Consider an alternative solution:
if (isBigEnough > 10) {
    return "num is greater than 10";
  } else {
    return "num is not big enough";